roborazzi-compose-preview-scanner-support/src/main/java/com/github/takahirom/roborazzi/RobolectricPreviewInfosApplier.kt

```diff
@@ -4,12 +4,22 @@ import android.content.res.Configuration
 import org.robolectric.RuntimeEnvironment.setFontScale
 import org.robolectric.RuntimeEnvironment.setQualifiers
 import sergio.sastre.composable.preview.scanner.android.AndroidPreviewInfo
+import sergio.sastre.composable.preview.scanner.android.device.domain.RobolectricDeviceQualifierBuilder
 import sergio.sastre.composable.preview.scanner.core.preview.ComposablePreview
 
 @ExperimentalRoborazziApi
 fun ComposablePreview<AndroidPreviewInfo>.applyToRobolectricConfiguration() {
   val preview = this
 
+  fun setDevice(device: String){
+    if (device.isNotBlank()) {
+      RobolectricDeviceQualifierBuilder.build(device)?.run {
```

Comment:
Users may encounter compiler errors when updating Roborazzi. Since version management in Roborazzi Gradle Plugin can be challenging, how about adding a comment like this:

```suggestion
      // Requires `io.github.sergio-sastre.ComposablePreviewScanner:android:0.4.0` or later
      RobolectricDeviceQualifierBuilder.build(device)?.run {      ```